pages/demo-get-involved.vue

```diff
@@ -230,9 +367,53 @@
     </div>
   </form>
 </template>
+<script>
+// import { createApp } from 'vue'
+// var firstName = ''
+// var lastName = ''
+// var email = ''
+
+// var reasonOfJoining = ''
+// var city = ''
+// var state = ''
+// var zipCode = ''
+// var country = ''
+// const myObject = new createApp({
+//   el: '.container',
+//   data: { firstName: 'xyz' },
+// })
+// myObject.mount('.container')
+import Airtable from 'airtable'
+var base = new Airtable({ apiKey: 'keyQ54SEzABIZEm9F' }).base(
+  'appv6M4U4ckXcHzhU'
+)
+export default {
+  name: 'formUI',
+  data() {
+    return {
+      name: null,
+      phone: null,
+
+      email: null,
+      linkedin: null,
+      instagram: null,
+      facebook: null,
+      telegram: null,
+      reasonOfJoining: null,
+      detailOfHelp: null,
+    }
+  },
+  methods: {
+    submit() {
+      console.warn('values : ', this.name)
```

Comment:
console and debug code shouldn't be in "Pull requests ready for review"

Reply:
Oh this is second time around, sorry, being a newbie thing!